packages/modular-scripts/src/getModularRoot.ts

```diff
@@ -1,4 +1,3 @@
-// keep this in sync with modular-views.macro/src/getModularRoot
 import * as path from 'path';
```

Comment:
I feel like we should just expose this API as we’ll end up needing it in multiple places anyway - or it should just be a stand-alone that we can share.

Reply:
I think It's fine to duplicate this code, particularly because it's used differently at different spots, so they're not actually in sync.